industrial_ci/src/tests/source_tests.sh

```diff
@@ -97,6 +97,92 @@ function run_clang_tidy_check {
     fi
 }
 
+function ici_combine_cpp_reports {
```

Comment:
these functions should go into a special file, like `coverage.sh`.

Reply:
Where should I source `coverage.sh`, under `ci_main.sh` or `source_tests.sh`?

Reply:
`ci_main.sh` for now.
Even the upload function should go in there.